pkgs/top-level/all-packages.nix

```diff
@@ -13034,6 +13034,8 @@ with pkgs;
 
   man-pages = callPackage ../data/documentation/man-pages { };
 
+  material-icons-ttf = callPackage ../data/fonts/material-icons-ttf { };
```

Comment:
Agreed, I think it should be removed both from the directory and the entry in all-packages.